Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR ${HTML_DIR}
 COPY docker-entrypoint.sh .
 # This will be generated in the pipeline.
 # Comment out if building locally
-COPY instance.sbom.json /opt/telicent/sbom.json
+COPY instance.sbom.json /opt/telicent/sbom/instance.sbom.json
```

Comment:
I thought there was a specific naming convention for this file. Admittedly it was in the ticket the python guys were using for their workflows but aren't we trying to stay as consistent as possible?

Reply:
Sure could be; This is not knowledge I've been given.  
I'll change it to sbom.json

Reply:
Must be directory, no requirement on name. See: https://telicent.atlassian.net/wiki/spaces/DT/pages/26738689/Vulnerability+scanning#%F0%9F%8E%AF-Scope